verifier.sh

```diff
@@ -467,6 +470,12 @@ export GEM_TIME_INVARIANT_OUTPUTS=y
 export NUMBA_DISABLE_JIT=1
 
 # run webui
+echo \$GEM_TOOLS_ONLY
+if [ -z \$GEM_TOOLS_ONLY ]; then
+    cd oq-engine/openquake/server
+    cp local_settings.py.tools.tmpl local_settings.py
```

Comment:
Please refer to local_settings.py.tools (without tmpl)

Reply:
Solved with this commit here: https://github.com/gem/oq-platform-standalone/pull/55/commits/c630b09996b2a4523ecdc59fbd941ce4146d3ff6